infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntime.java

```diff
@@ -150,6 +173,29 @@ protected void internalStop(Map<String, String> stopOptions) throws Infrastructu
     return emptyMap();
   }
 
+  private void registerAbnormalStopHandler() throws InfrastructureException {
```

Comment:
Sorry, but I get lost in tests a bit, can you point me to the tests that are covering this code?

Reply:
done